src/controllers/personsEmailController.ts

```diff
@@ -0,0 +1,61 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../config";
+import { BASE_URL, PERSONAL_CODE, EMAIL_ADDRESS, DATE_OF_BIRTH } from "../types/pageURL";
+import { addLangToUrl, getLocaleInfo, getLocalesService, selectLang } from "../utils/localise";
+import { formatValidationError, getPageProperties } from "../validations/validation";
+import { validationResult } from "express-validator";
+import { Session } from "@companieshouse/node-session-handler";
+import { ClientData } from "model/ClientData";
+import { USER_DATA } from "../utils/constants";
+import { saveDataInSession } from "../utils/sessionHelper";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const session: Session = req.session as any as Session;
+    const clientData: ClientData = session.getExtraData(USER_DATA) ? session.getExtraData(USER_DATA)! : {};
```

Comment:
I would suggest not to do this. Just get the session data and then later suer clientData? while doing dot operation